systems/summit_user_guide.rst

```diff
@@ -2001,6 +2001,8 @@ discussion of the ``cpu_index_using`` control and its interaction with various
 SMT modes.
 
 
+.. note::
+    An error is reported when using ERF on Summit. Please visit `Known Issues <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#open-issues>`__ for solutions. 
```

Comment:
Should we give more detail here?  

Please note, a known bug is currently preventing execution of most ERF use cases.  We are working to resolve the issue.  If you experience issues using the ERF feature, please see the work around in `Known Issues <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#open-issues>` .

Reply:
Thanks. Updated